### PR TITLE
BM-577 Fix app version number on settings views

### PR DESCRIPTION
**Pull request by pkozielecki**
### Ticket
https://netguru.atlassian.net/browse/BM-577
 
 
### Task Description
Fix app version number shown on settings views
 
 
### Aditional Notes (optional)
-

### CODE REVIEW

File: Baby Monitor/Source Files/Modules/Settings/BaseSettingsView.swift [hunk 1]
```diff
@@ -9,21 +9,24 @@ import RxSwift
 import RxCocoa
 
 class BaseSettingsView: UIView {
-    
+
+    lazy var appVersionProvider: AppVersionProvider = DefaultAppVersionProvider()
```
Comment (anyashka) [inline Baby Monitor/Source Files/Modules/Settings/BaseSettingsView.swift]: What about `AppDependencies`? :)

File: Baby Monitor/Source Files/Utils/AppVersionProvider.swift [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        return "\(appVersion) (\(buildNumber))"
+    }
+}
+
+protocol BundleDataProvider: class {
+    var buildNumber: String { get }
+    var bundleVersion: String { get }
+}
+
+extension Bundle: BundleDataProvider {
```
Comment (anyashka) [inline Baby Monitor/Source Files/Utils/AppVersionProvider.swift]:
```suggestion
extension Bundle: BundleDataProvider { }
```

File: Baby Monitor/Source Files/Modules/Settings/BaseSettingsView.swift (hunk 1 above)
Comment (pkozielecki) [reply Baby Monitor/Source Files/Modules/Settings/BaseSettingsView.swift]: 👍

File: Baby Monitor/Source Files/Utils/AppVersionProvider.swift (hunk 2 above)
Comment (pkozielecki) [reply Baby Monitor/Source Files/Utils/AppVersionProvider.swift]: 👍

**Pull request merged**